File: src/main/java/com/divide2/dic/model/Label.java
package com.divide2.dic.model;

import lombok.Data;
import org.springframework.data.elasticsearch.annotations.Document;

import javax.persistence.*;

/**
 * Created by bvvy on 2018/1/4.
 * com.divide2.dic.model
 */
@Table(name = "sys_label")
@Entity
@Data
@Document(indexName = "labels",type = "label",refreshInterval = "-1")
public class Label {

    @Id
    @GeneratedValue
    @org.springframework.data.annotation.Id
    private Integer id;

    @Column(name = "cn_label")
    private String cnLabel;

    private String code;

}

File: src/main/kotlin/com/divide2/label/Label.kt
package com.divide2.label

import javax.persistence.*

/**
 * Created by bvvy on 2018/1/4.
 * com.divide2.dic.model
 */
@Table(name = "sys_label")
@Entity
class Label(
        @Id
        @GeneratedValue
        var id: Int = 0,

        @Column(name = "cn_label")
        var cnLabel: String = "",

        var code: String = ""
)
